plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreViewBase.java

```diff
@@ -155,18 +140,28 @@ public String getObjectDefinitionText(DBRProgressMonitor monitor, Map<String, Ob
         return ddl.toString();
     }
 
+    private void fetchViewQueryResult(JDBCSession session) throws SQLException, DBException {
```

Comment:
It's used in just a single place. Why extract it?

Reply:
By my opinion it easy to read the host function

Reply:
Debatable